presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -36,7 +36,6 @@
 #include "presto_cpp/main/operators/PartitionAndSerialize.h"
 #include "presto_cpp/main/operators/ShuffleRead.h"
 #include "presto_cpp/main/operators/UnsafeRowExchangeSource.h"
-#include "presto_cpp/main/types/PrestoToVeloxConnector.h"
```

Comment:
Thanks for cleaning up unused includes

Reply:
This header is used by this file. Why remove it?